PWGHF/vertexingHF/AliAnalysisTaskHFSimpleVertices.cxx

```diff
@@ -1139,6 +1308,49 @@ Int_t AliAnalysisTaskHFSimpleVertices::DzeroSkimCuts(AliAODRecoDecayHF2Prong* ca
   if(isD0bar) returnValue+=2;
   return returnValue;
  
+}
+//______________________________________________________________________________
+Int_t AliAnalysisTaskHFSimpleVertices::JpsiSkimCuts(AliAODRecoDecayHF2Prong* cand)
+{
+  bool isJpsi = true;
+  Double_t ptCand = cand->Pt();
+  if (ptCand < fJpsiSkimCuts[0]) return 0;
+  Double_t m0=cand->InvMassJPSIee();
+  if(m0 < fJpsiSkimCuts[1] || m0 > fJpsiSkimCuts[2]) isJpsi=false;
```

Comment:
I think you can avoid using isJpsi and just do 
if(m0 < fJpsiSkimCuts[1] || m0 > fJpsiSkimCuts[2]) return 0;

Below you can then replace:
  Int_t returnValue=0;
  if(isJpsi) returnValue+=1;
  return returnValue;
with a simple: 
  return 1;
This would allow you to spare the extra if conditions at line 1320 and 1324